docs/security/docs/03-04-roles-in-kyma.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Roles in Kyma
+type: Details
+---
+
+Kyma uses roles and groups to manage access in the cluster. Every cluster comes with 5 predefined roles which give the assigned users different level of permissions suitable for different purposes.
+These roles are defined as ClusterRoles and use the Kubernetes mechanism of aggregation, which allows to combine multiple ClusterRoles into a single ClusterRole. Using the aggregation mechanism allows to efficiently manage access to Kubernetes resources and Kyma-specific resources.
+
+>**NOTE:** Read [this](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles) Kubernetes documentation to learn more about the aggregation mechanism used to define Kyma roles.
+
+You can assign any of the predefined roles to a user or to a group of users in the context of:  
+  - the entire cluster, creating a [ClusterRoleBinding](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#rolebinding-and-clusterrolebinding)
+  - a specific Namespace, creating a [RoleBinding](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#rolebinding-and-clusterrolebinding)
+
+The predefined roles, arranged in the order of increasing access level, are:
+
+| Role | Description |
+| --- | --- |
+| **kyma-essentials** | The basic role required to allow the users to see the Console UI of the cluster. This role doesn't give the user rights to manipulate any resources. |
+| **kyma-view** | The role allowing to get and list Kubernetes resources and Kyma-specific resources. |
```

Comment:
```suggestion
| **kyma-view** | The role allowing to list Kubernetes and Kyma-specific resources. |
```

Reply:
"get" and "list" are kubernetes resources and that's why they're both listed here

Reply:
The reason I removed `get`  was that it was used as a description here, not as a command. Thus, in the overall meaning, isn't it just about listing their details (read access permissions as the opposite to write access permissions)?